contracts/mocks/Stateless.sol

```diff
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: MIT
 
-pragma solidity ^0.8.24;
+pragma solidity ^0.8.26;
```

Comment:
Shouldn't be 25?

```suggestion
pragma solidity ^0.8.25;
```

Reply:
`mocks/Stateless.sol` imports `utils/draft-InteroperableAddress.sol` with itself does `require(<condition>, <error>);`

This format of require was introduced in [0.8.26](https://github.com/ethereum/solidity/blob/develop/Changelog.md#0826-2024-05-21)

Reply:
Ah, got confused between 0.8.27 and 0.8.26. Makes sense